### PR TITLE
Refactor/pregrasp planner

### PR DESCRIPTION
**Pull request by abhishekpadalkar**
@sthoduka Please review the changes.

### CODE REVIEW

File: mcr_manipulation/mcr_manipulation_pose_selector/ros/src/mcr_manipulation_pose_selector_ros/reachability_pose_selector_class.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        self.linear_offset = None
+
+        # kinematics class to compute the inverse kinematics
+        self.kinematics = kinematics.Kinematics(self.arm)
+
+        # Time allowed for the IK solver to find a solution (in seconds).
+        self.ik_timeout = 0.5
+
+        # node cycle rate (in hz)
+        self.loop_rate = rospy.Rate(rospy.get_param('~loop_rate', 10.0))
```
Comment (sthoduka) [inline mcr_manipulation/mcr_manipulation_pose_selector/ros/src/mcr_manipulation_pose_selector_ros/reachability_pose_selector_class.py]: no need for loop rate

**Comment by sthoduka** [comment]
Can you run roslint tests?
`catkin build PACKAGE_NAME --no-deps --make-args roslint`

**Comment by abhishekpadalkar** [comment]
roslint test appears to be working for both the packages.

**Comment by abhishekpadalkar** [comment]
These packages are needed to be tested on indigo.

**Pull request merged**